tests/test_mc_simulator.py

```diff
@@ -190,10 +190,15 @@ def test_MantarrayMCSimulator_read__returns_empty_bytes_if_no_bytes_to_read(
     assert actual_item == expected_item
 
 
-def test_MantarrayMCSimulator_write__puts_object_into_input_queue(
-    mantarray_mc_simulator,
-):
-    input_queue, _, _, _, simulator = mantarray_mc_simulator
+def test_MantarrayMCSimulator_write__puts_object_into_input_queue__with_no_sleep_after_write():
```

Comment:
if the title is `with_no_sleep_after_write`, should there be an assertion that sleep isn't being called?